src/Sylius/Component/Core/Model/Order.php

```diff
@@ -460,11 +466,16 @@ public function getNonDiscountedItemsTotal(): int
         return $total;
     }
 
-    public function getCreatedByGuest(): bool
+    public function isCreatedByGuest(): bool
     {
         return $this->createdByGuest;
     }
 
+    public function getCreatedByGuest(): bool
```

Comment:
I'm not sure we should introduce all these methods in the model. It has been done in 1.12 already and e.g. `getCreatedByGuest` is deprecated.

Reply:
I don't know what we should do. This Pr is only adding changes from: https://github.com/Sylius/Sylius/pull/13968/files and merge conflicts. This part is deprecated in 1.12 so we can deprecate it also in 1.11  or just close this PR and don't add these changes to 1.11

Reply:
I would be for using `setCreatedByGuest` in 1.11 instead of moving changes from 1.12 to 1.11. So we don't need to deprecate anything and do the fix by setting `Order::createdByGuest` correctly. This PR https://github.com/Sylius/Sylius/pull/14673 seems to me to be a better one.

Reply:
But PR that you linked (#14673) is adding changes to 1.12 and in 1.12 we already have changes from this PR right here and if I understand this comment correctly: https://github.com/Sylius/Sylius/pull/14007#issuecomment-1140484047
We should first add logic to 1.11 from Łukasz PR

Reply:
Ok, now I see there are two ways on solving the problem. The first one is https://github.com/Sylius/Sylius/pull/14673 and the second one is to copy changes from 1.12. As this PR is about copying changes I'm resolving the comment :grinning: